common/src/types/trpc.ts

```diff
@@ -0,0 +1,139 @@
+import type { TRPCClientErrorLike } from "@trpc/react";
+import type { Procedure } from "@trpc/server/dist/declarations/src/internals/procedure";
+import type {
+  Router,
+  ProcedureRecord,
+} from "@trpc/server/dist/declarations/src/router";
+import type { Subscription } from "@trpc/server/dist/declarations/src/subscription";
+import type { TRPCClient as _TRPCClient } from "@trpc/client";
+import { UseMutationResult, UseQueryResult } from "react-query";
+
+import type { AppRouter } from "../../../backend/src/router";
+
+type AnyRouter<TContext = any> = Router<any, TContext, any, any, any, any, any>;
```

Comment:
This one should be an issue in `trpc/trpc` repo as most of these types are already implemented but not exported for some reason.